File: python_programming/practices/time_of_day.py
# ME 6th Time of Day code

import datetime

current = datetime.datetime.now()

hour = current.hour

if hour <= 12:
    print(f"Looks like its {hour} good morning!")
elif hour <= 18:
    print(f"Hello there looks like the time is {hour} good afternoon!")
else:
    print(f"Looks like its {hour} good afternoon!")

File: c_programming/programming/time_of_day.c
// ME 6th Time of Day
#include <stdio.h>
#include <string.h>

int main(void){
    int time;
    printf("Hello what is the time in Military: \n");
    scanf("%d", &time);

    if(time <= 12){
        printf("Good Morning!");
    }
    else if(time <= 20){
        printf("Good evening!");
    }
    else{
        printf("Good Night!");
    }
    return 0;
}